build.gradle.kts

```diff
@@ -89,3 +93,41 @@ dokkatoo {
         }
     }
 }
+
+tasks.withType<LockCopyTask>().configureEach {
```

Comment:
Wondering if we haven't reach the tipping point where we just pragmatically use the default yarn.